packages/nimble-blazor/NimbleBlazor/Components/NimbleTableColumnNumberText.razor.cs

```diff
@@ -54,8 +54,14 @@ public partial class NimbleTableColumnNumberText : NimbleTableColumn, IFractiona
     public NumberTextAlignment? Alignment { get; set; }
 
     /// <summary>
-    /// The number of digits to display after the decimal place when the column's format is decimal.
+    /// The number of digits to display after the decimal place when the column's format is decimal. Mutually exclusive with DecimalMaximumDigits.
```

Comment:
It might be helpful to make it clear that setting both `DecimalDigits` and `DecimalMaximumDigits` results in an invalid state (instead of just saying "Mutually exclusive" for both.